readme.md

```diff
@@ -358,6 +358,7 @@
 - [Construct 2](https://github.com/WebCreationClub/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
+- [Game Datasets](https://github.com/leomaurodesenv/game-datasets#readme) - Tools for Artificial Intelligence in Games.
```

Comment:
`Games` => `games` and I think the description could still be clearer. `Tools` should also not be there as the list contains more than just tools.

Reply:
"Materials and datasets for Artificial Intelligence in games."

Reply:
Yes, better.